sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql

```diff
@@ -374,9 +374,9 @@ main_ping AS (
     ARRAY_AGG(normalized_os_version RESPECT NULLS ORDER BY submission_date)[
       SAFE_OFFSET(0)
     ] AS normalized_os_version,
-    CAST(
-      NULL AS STRING
-    ) AS startup_profile_selection_reason, -- main_v5:payload.processes.parent.scalars.startup_profile_selection_reason
+    ARRAY_AGG(startup_profile_selection_reason_first RESPECT NULLS ORDER BY submission_date)[
+      SAFE_OFFSET(0)
+    ] AS startup_profile_selection_reason,
```

Comment:
Given the discussion in the doc, should we call this
```suggestion
    ] AS first_reported_startup_profile_selection_reason,
```

Reply:
I think that applies to all columns in this table no? They should all be the first reported value

Reply:
hah, I guess so. I would have said to name that all that way if I had realized